### ISSUE TITLE
Cannot adjust mod name size

### ISSUE THREAD

**Issue by HoodBlah** [PrismLauncher/PrismLauncher#3189]
### Operating System

Windows

### Version of Prism Launcher

9.1

### Version of Qt

6.7.3

### Description of bug

In the mod list, I cannot adjust the name title bar size at the top. I can drag all the other bars to make them bigger or smaller but the name (probably the most important one) wont move.

### Steps to reproduce

Try to change the size of the name title

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by HoodBlah** [comment]
I played around with it a bit more and it seems to be something with the window size and how many categorizes are enabled.

**Comment by Trial97** [comment]
You should but not by resizing the 'Name' column but by resizing adjacent columns (some qt stuff regarding the fact that the view there needs a stretch column to fill the dialog)

**Comment by HoodBlah** [comment]
That doesn't make sense, if I can change the size of all the other column directly then why can't I change the size of the name column directly

**Comment by Trial97** [comment]
Again this is my statement: `qt is at fault here because of how column stretching is implemented`

**Comment by HoodBlah** [comment]
Still an issue

**Comment by LichLord91** [comment]
still an issue

**Comment by Ikethepro18** [comment]
This issue looks to also affect the 'Side' column, but none others

**Comment by Eldrinn-Elantey** [comment]
The launcher picks the stretch section, not Qt: ModFolderModel sets Name to QHeaderView::Stretch (launcher/minecraft/mod/ModFolderModel.cpp:73), and ModListView::setModel does the same for the version list. Would you accept moving the stretch to the last section so Name resizes like the rest?

**Comment by Trial97** [comment]
If we move the stretch to last one then the last column would fill the table view. Like I'm not against fixing this: but find a way where the name is couping more space than other columns, otherwise as an example you will have the last column that most probably only contain a number or a small information extending too much

**Comment by Eldrinn-Elantey** [comment]
Qt has no mode that stretches a section and still lets you drag it, so the spare width has to go to Name by hand: keep every section Interactive and add the viewport delta to the Name section in resizeEvent, with a minimum width so it cannot collapse. Does that match what you have in mind?

**Comment by Trial97** [comment]
yes it sounds like a plan, a weird one but a plan never the less.
I would be satisfied if the name column acts as the principal column(taking enough space to fill the table if the other columns are too small). 
So the other columns can't occupy more space than name, and the width of all columns should occupy all the table space regardless of the window size.

**Comment by Eldrinn-Elantey** [comment]
With the table always filled, widening Version still takes the width from Name, so the grabbed handle stays put and Version grows to the left, which is the part that bothers users here. Would you accept a horizontal scrollbar when the columns exceed the view, and should "can't occupy more space than name" apply to each column or to all of them together?

**Comment by Trial97** [comment]
Sadly that looks horrible for me(I tried it) so I would not accept that.

**Comment by Eldrinn-Elantey** [comment]
I'll open a PR where each handle moves width between its two neighbours: the table stays filled and Name only takes up window resizes. Let's settle the details there.

**Comment by Eldrinn-Elantey** [comment]
I'll open a PR where each handle moves width between its two neighbours: the table stays filled and Name only takes up window resizes. Let's settle the details there.

**Comment by HoodBlah** [comment]
Sorry if I've misunderstood, but can't you just let the columns be whatever size and just add a side scroller? That way you don't need to worry about window size or restrictions or anything no?
